Scripts/Tracking/Follow/Modifier/TransformSourceFollowActiveTarget.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace VRTK.Core.Tracking.Follow.Modifier
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// The TransformSourceFollowActiveTarget will force the transformations of the source Transform to match those of the target Transform.
+    /// </summary>
+    public class TransformSourceFollowActiveTarget : FollowModifier
+    {
+        /// <summary>
+        /// The ProcessFirstAndActiveOnly method determines whether the FollowModifier should process all targets or just the first active target.
+        /// </summary>
+        /// <returns>Always returns `true`.</returns>
+        public override bool ProcessFirstAndActiveOnly()
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// The UpdatePosition method attempts to set the source Transform position to the target Transform position.
+        /// </summary>
+        /// <param name="source">The source to modify.</param>
+        /// <param name="target">The target to utilise in the modification.</param>
+        public override void UpdatePosition(Transform source, Transform target)
+        {
+            if (source != null && target != null)

```

Comment:
Shall we just silently do nothing, or is this something we should be throwing an `ArgumentException` for?

Reply:
I don't think throwing an exception is appropriate. If you don't pass anything then nothing should happen imho

passing two nulls isn't an exception, its just nothing will happen